src/pps/pps.proto

```diff
@@ -464,7 +464,9 @@ message PipelineInfo {
     Determined determined = 36;
   }
   Details details = 12;
-  string details_json = 13;
+  reserved "details_json";
+  string user_spec_json = 13;
```

Comment:
Could you either comment these fields like you did in `CreatePipelineTransaction` or have a `see: CreatePipelineTransaction` if you don't want to duplicate?